src/app/services/models/annotation.service.ts

```diff
@@ -0,0 +1,74 @@
+import { Inject, Injectable } from "@angular/core";
+import { AudioRecordingsService } from "@baw-api/audio-recording/audio-recordings.service";
+import { BawSessionService } from "@baw-api/baw-session.service";
+import { TagsService } from "@baw-api/tag/tags.service";
+import { AudioEvent } from "@models/AudioEvent";
+import { AudioRecording } from "@models/AudioRecording";
+import { Annotation } from "@models/data/Annotation";
+import { API_ROOT } from "@services/config/config.tokens";
+import { firstValueFrom } from "rxjs";
+
+@Injectable()
+export class AnnotationService {
+  public constructor(
+    private tagsApi: TagsService,
+    private audioRecordingsApi: AudioRecordingsService,
+    private session: BawSessionService,
+    @Inject(API_ROOT) private apiRoot: string
+  ) {}
+
+  public async buildAnnotation(audioEvent: AudioEvent): Promise<Annotation> {
```

Comment:
todo: i'd like to see this class get an API like our other services. 

`show`, `create`, and `update`